.github/workflows/schedule_zoom_talks.yml

```diff
@@ -0,0 +1,27 @@
+name: Schedule Zoom Talks
+on:
+  repository_dispatch:
+    types: [schedule-zoom-talk]
+    
+jobs:
+  scheduleTalks:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+        
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f bots/zoom_scheduling_bot_requirements.txt ]; then pip install -r bots/zoom_scheduling_bot_requirements.txt; fi
```

Comment:
Note to self: Should not forget to update requirements

Reply:
I propose to use a single requirements file. Also the if statement is superfluous: we know that the requirements exist.